CRUD_functions/cohort/cohort.js

```diff
@@ -0,0 +1,93 @@
+const knex = require('knex');
+const router = require('express').Router();
+
+const KnexConfig = {
+    client: 'sqlite3',
+    connection: {
+        filename: './data/lambda.db3'
+    },
+    useNullAsDefault: true,
+    debug: true
+}
+
+const db = knex( KnexConfig );
+
+//GET ALL COHORTS
+router.get( '/', (req , res) => {
+    db( 'cohort' )
+        .then( cohorts =>
+            res.status( 200 ).json( cohorts ) 
+        )
+        .catch( error => res.status( 500 ).json( error ))
+});
+
+//GET INDIVIDUAL COHORTS
+router.get('/:id', (req, res) => {
+    db( 'cohort' )
+        .where({ id: req.params.id })
+        .first()
+        .then( cohort => {
+            if ( cohort ) {
+                res.status( 200 ).json( cohort )
+            } else {
```

Comment:
the else here should be returning a 404 not found, the catch is where the 500 error response is